assets/src/tests/unit/index.test.ts

```diff
@@ -0,0 +1,14 @@
+// TO BE REMOVED, this is just for example purposes.
+const sum = (...a: number[]) => a.reduce((acc, val) => acc + val, 0);
+
+test('basic', () => {
+	expect(sum()).toBe(0);
+});
+
+test('basic again', () => {
+	expect(sum(1, 2)).toBe(3);
+});
+
+test('basic async', async () => {
+	expect(sum(1, 2)).toBe(3);
+}, 1000 /* optional timeout */);
```

Comment:
Cool!

Reply:
Yeah, testing Apollo queries/ mutation will be easier with async support in Jest.